### PR TITLE
Fix open ordres

### PR DESCRIPTION
**Pull request by dr497**

### CODE REVIEW

File: src/pages/OpenOrdersPage.tsx [hunk 1]
```diff
@@ -6,10 +6,13 @@ import {
...
 } from '../utils/markets';
 import OpenOrderTable from '../components/UserInfoTable/OpenOrderTable';
-import { Button } from 'antd';
+import { Button, Row } from 'antd';
 import { OrderWithMarketAndMarketName } from '../utils/types';
+import { useWallet } from '../utils/wallet';
+import WalletConnect from '../components/WalletConnect';
 
 export default function OpenOrdersPage() {
+  const { connected } = useWallet(0);
```
Comment (nathanielparke) [inline src/pages/OpenOrdersPage.tsx:15]: `useWallet` doesn't take any arguments as it is currently written. Is this a typo maybe?

File: src/utils/markets.tsx [hunk 2]
```diff
@@ -787,67 +795,56 @@ export function useAllOpenOrdersBalances() {
...
+        setLoaded(false);
+        const _openOrders: { orders: Order[]; marketAddress: string }[] = [];
+        for (let i = 0; i < USE_MARKETS.length; i++) {
+          const market = await Market.load(
+            connection,
+            USE_MARKETS[i].address,
+            undefined,
+            USE_MARKETS[i].programId,
+          );
+          const orders = await market.loadOrdersForOwner(
```
Comment (nathanielparke) [inline src/utils/markets.tsx:827]: Let's add a `cacheDurationMs` parameter here which will cut down on the number of times we call `getProgramAccounts` which is a non-performant query for validators.

Comment (nathanielparke) [reply src/utils/markets.tsx:827]: 30 seconds seems reasonable to me.

**Review by nathanielparke — CHANGES_REQUESTED** [review]

File: src/pages/OpenOrdersPage.tsx (hunk 1 above)
Comment (dr497) [reply src/pages/OpenOrdersPage.tsx:15]: Yep typo! Thanks for the catch

File: src/utils/markets.tsx (hunk 2 above)
Comment (dr497) [reply src/utils/markets.tsx]: Done!

**Pull request merged**